apps/expo/package.json

```diff
@@ -132,7 +132,7 @@
     "domain-browser": "^4.22.0",
     "dotenv": "^10.0.0",
     "events": "^3.3.0",
-    "expo": "next",
+    "expo": "~46.0.7",
```

Comment:
yeah, patch files will have some issues when the lib version has changed if we use `next`. 

for example: `react-native+0.69.3.patch` file, the expo `next` version is used react-native `v0.69.4`, If we forget to update this file, this patch will not be used.

Reply:
so I'm not sure if `expo` `~46.0.7` uses` react-native v0.69.4`.

Reply:
good catch, will update the patch!

> so I'm not sure if expo ~46.0.7 uses react-native v0.69.4.
yeah, it's using 0.69.4, can confirm!